src/utils/time-entry-utils/ContextTimeEntry.js

```diff
@@ -39,6 +48,11 @@ export class ContextTimeEntry {
     return this;
   }
 
+  setFinishDate(finishDate) {
+    this.finishDate = finishDate;
+    return this;
+  }
```

Comment:
Do we need a finish date? `finishTime` is a dateTime, so we previously decided to get rid of it